### PR TITLE
Feature/port map file nodes

### PR DESCRIPTION
**Pull request by adev4a**
# PR Details
This PR ports nodes for vector map loader, vector map visualizer, pcd map loader and georeference param to ros2.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: common/map_file_ros2/config/parameters.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+update_rate: 0
```
Comment (adev4a) [inline common/map_file_ros2/config/parameters.yaml]: Should this package have a param file? In the ros1 implementation the values for the params in these nodes comes from carma. The same approach is used here, the parameter values are directly passed in carma-config and platform, 
having a config file creates a false sense that the parameter values can be controlled from here.

Also there are 4 nodes within this package with different launch files. If we do want to keep the param yml, we need to add params for all the nodes.
The ones specified here are only for the points_map_loader node.

File: common/map_file_ros2/test/test_get_transform.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2019-2021 LEIDOS.
```
Comment (MishkaMN) [inline common/map_file_ros2/test/test_get_transform.cpp]: year change

File: common/map_file_ros2/package.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  <depend>rclcpp_components</depend>
+  <depend>std_msgs</depend>
+  <depend>geometry_msgs</depend>
+  <depend>curl</depend>
+  <depend>libpcl-all-dev</depend>
+
+  <depend>tf2_geometry_msgs</depend>
+  <depend>tf2_ros</depend>
+  <depend>tf2</depend>
+  <depend>vector_map</depend>
```
Comment (MishkaMN) [inline common/map_file_ros2/package.xml]: this is a ros1 package, it doesn't seem to build with it. Do we need it?

**Review by MishkaMN — CHANGES_REQUESTED** [review]

File: common/map_file_ros2/config/parameters.yaml (hunk 1 above)
Comment (MishkaMN) [reply common/map_file_ros2/config/parameters.yaml]: I don't think we need it if we are not using it

File: common/map_file_ros2/package.xml (hunk 3 above)
Comment (adev4a) [reply common/map_file_ros2/package.xml]: It looks like that package is not required, in the ros1 version of the package there is an additional node 'vector_map_loader' which uses it but we are not porting that to ros2

File: common/map_file_ros2/config/parameters.yaml (hunk 1 above)
Comment (adev4a) [reply common/map_file_ros2/config/parameters.yaml]: Removed this params file. Parameters will need to be passed as arguments when launching the node

**Review by MishkaMN — APPROVED** [review]

**Pull request merged**